library/js/jquery-3.4.0.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.4.0 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
This is not needed now in the codbase. The standard version should be `3.3.1` as you called in `call_required_libraries(...`

Reply:
Drop(delete) this.